integreat_cms/api/v3/events.py

```diff
@@ -2,13 +2,13 @@
 This module includes functions related to the event API endpoint.
 """
 
-from datetime import timedelta
+from datetime import datetime, timedelta
 
 from django.conf import settings
 from django.http import JsonResponse
 from django.utils import timezone
 from django.utils.html import strip_tags
-
+from django.utils import timezone
```

Comment:
```suggestion

```
`timezone` is already imported in line 9

Reply:
Wait my IDE still marked it as missing. Maybe something was still loading, my bad :D